.golangci.yml

```diff
@@ -197,7 +197,7 @@ linters:
       # Report about not checking of errors in type assertions: `a := b.(MyStruct)`.
       # Such cases aren't reported by default.
       # Default: false
-      check-type-assertions: true
+      check-type-assertions: false
```

Comment:
Why so? Not doing it and making a mistake results in a panic... this is kind of dangerous?